tools/webpack-plugin/src/BacktracePlugin.ts

```diff
@@ -52,45 +51,60 @@ export class BacktracePlugin implements WebpackPluginInstance {
             logger.log(`received ${entries.length} files for processing`);
 
             for (const [asset, sourcePath, sourceMapPath] of entries) {
-                const stats: AssetStats = {};
-                assetStats.set(asset, stats);
-
                 let debugId: string;
 
                 logger.time(`[${asset}] process source and sourcemap`);
                 try {
                     debugId = await this._sourceProcessor.processSourceAndSourceMapFiles(sourcePath, sourceMapPath);
-                    stats.debugId = debugId;
-                    stats.processSource = true;
+                    processResults.set(asset, debugId);
                 } catch (err) {
                     logger.error(`[${asset}] process source and sourcemap failed:`, err);
-                    stats.processSource = err instanceof Error ? err : new Error('Unknown error.');
+                    processResults.set(asset, err instanceof Error ? err : new Error('Unknown error.'));
                     continue;
                 } finally {
                     logger.timeEnd(`[${asset}] process source and sourcemap`);
                 }
+            }
 
-                if (!this._sourceMapUploader) {
-                    logger.log(`[${asset}] file processed`);
-                    continue;
-                }
-
-                logger.time(`[${asset}] upload sourcemap`);
+            if (this._sourceMapUploader) {
+                logger.time(`upload sourcemaps`);
                 try {
-                    const result = await this._sourceMapUploader.upload(sourceMapPath, debugId);
-                    stats.sourceMapUpload = result;
-                    logger.log(`[${asset}] file processed and sourcemap uploaded`);
+                    const archive = new ZipArchive();
+                    const request = this._sourceMapUploader.uploadSymbol(archive);
+
+                    for (const [asset, , sourceMapPath] of entries) {
```

Comment:
Is this syntax just ignoring the second item of entries? Could we use an underscore or something?

Reply:
Yeah, exactly for that. The underscore isn't really necessary there and it generates some ESLint warnings:
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/9ada51c1-f660-40ae-a7a3-80a8c420ba63)